src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console */
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  today: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+      console.info(this.state.today.toUTCString().slice(-12, -4));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props, prevState: State) {
+    if (prevState.today.getSeconds() === this.state.today.getSeconds()) {
```

Comment:
Didn't really understand this, we were about to check if the name has changed, not time